src/modlunky2/ui/trackers/runstate.py

```diff
@@ -526,6 +530,30 @@ def update_is_chain(self):
         if self.win_state is WinState.TIAMAT:
             self.fail_chain()
 
+    def update_millionaire(self, inventory: Inventory):
+        collected = inventory.money + inventory.collected_money_total
```

Comment:
I think that these inventory attributes could potentially return None in which case we'd wanna defer on updating here.

Reply:
I'm unsure, but the checks are done elsewhere. I added them here.